specs/consensus.md

```diff
@@ -25,12 +27,29 @@ Consensus Rules
 
 ## Constants
 
-| name                   | type     | value   | unit   | description                                       |
-| ---------------------- | -------- | ------- | ------ | ------------------------------------------------- |
-| `NAMESPACE_ID_BYTES`   | `uint64` | `32`    | `byte` | Size of namespace ID, in bytes.                   |
-| `SHARE_SIZE`           | `uint64` | `256`   | `byte` | Size of transaction and message shares, in bytes. |
-| `SHARE_RESERVED_BYTES` | `uint64` | `1`     | `byte` | Bytes reserved at the beginning of each share.    |
-| `GENESIS_COIN_COUNT`   | `uint64` | `10**8` | `4u`   | `(= 100000000)` Number of coins at genesis.       |
+| name                             | type     | value   | unit    | description                                                                                                          |
+| -------------------------------- | -------- | ------- | ------- | -------------------------------------------------------------------------------------------------------------------- |
+| `NAMESPACE_ID_BYTES`             | `uint64` | `64`    | `byte`  | Size of namespace ID, in bytes.                                                                                      |
```

Comment:
Me thinking that it said "bits" instead of "bytes" and wanting to raise it from 32 bits to a more reasonable 64 bits. 32 bytes (256 bits) was perfectly fine. 😞